tool/tctl/common/resource_command.go

```diff
@@ -121,8 +121,8 @@ type ResourceCommand struct {
 	CreateHandlers map[ResourceKind]ResourceCreateHandler
 	UpdateHandlers map[ResourceKind]ResourceCreateHandler
 
-	// stdout allows to switch standard output source for resource command. Used in tests.
-	stdout io.Writer
+	// Stdout allows to switch standard output source for resource command. Used in tests.
+	Stdout io.Writer
```

Comment:
This change isn't necessary for this PR, but it's necessary for the upcoming integration tests. Long story short, we need to run tests and assert on their standard output from a different package. See https://github.com/gravitational/teleport.e/commit/bb9108fc169a0b439c4df420b0bfa310813fa838 to see how it's going to be used.